admin-manual/installation/installation.rst

```diff
@@ -758,8 +758,8 @@ above, follow these steps to complete the configuration of your new server.
 
 1. The Storage Service runs as a separate web application from the Archivematica
    dashboard. Go to the following link in a web browser and log in as user
-   *test* with the password *test*: http://localhost:8000 (or use the IP
-   address of the machine you have been installing on).
+   *test* with the password *test*: http://localhost:8000, or use the IP address
+   of the machine you have been installing on ( RPM packages use port 8001 ).
```

Comment:
Nitpick: typographical error, should read `(RPM packages use port 8001)` instead of `( RPM packages use port 8001 )`.